roles/hadoop/files/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Fixed in:
https://github.com/nlesc-sherlock/emma/commit/1a56f6d1e3fbb73d045b8f137e1239dd440c540f